[2023-02-10T23:08:17.101+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T10:20:00+00:00 [queued]>
[2023-02-10T23:08:17.107+0200] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T10:20:00+00:00 [queued]>
[2023-02-10T23:08:17.107+0200] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:08:17.107+0200] {taskinstance.py:1369} INFO - Starting attempt 1 of 4
[2023-02-10T23:08:17.107+0200] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T23:08:17.112+0200] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): extract_data_from_api> on 2023-01-10 10:20:00+00:00
[2023-02-10T23:08:17.117+0200] {standard_task_runner.py:52} INFO - Started process 16734 to run task
[2023-02-10T23:08:17.121+0200] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'test_process_sales', 'extract_data_from_api', 'scheduled__2023-01-10T10:20:00+00:00', '--job-id', '953', '--raw', '--subdir', 'DAGS_FOLDER/process_sales/mainS.py', '--cfg-path', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpy27hgz40', '--error-file', '/var/folders/1g/4w5x68gx3d92wgygb97jqbd40000gp/T/tmpgl33lbzi']
[2023-02-10T23:08:17.122+0200] {standard_task_runner.py:80} INFO - Job 953: Subtask extract_data_from_api
[2023-02-10T23:08:17.151+0200] {task_command.py:371} INFO - Running <TaskInstance: test_process_sales.extract_data_from_api scheduled__2023-01-10T10:20:00+00:00 [running]> on host admins-MacBook-Air-2.local
[2023-02-10T23:08:17.179+0200] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=admin@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_process_sales
AIRFLOW_CTX_TASK_ID=extract_data_from_api
AIRFLOW_CTX_EXECUTION_DATE=2023-01-10T10:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-10T10:20:00+00:00
[2023-02-10T23:08:17.467+0200] {python.py:173} INFO - Done. Returned value was: {'message': 'Authorization token not set properly. You are not authorised to use this resourse'}
[2023-02-10T23:08:17.479+0200] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=test_process_sales, task_id=extract_data_from_api, execution_date=20230110T102000, start_date=20230210T210817, end_date=20230210T210817
[2023-02-10T23:08:17.496+0200] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10T23:08:17.509+0200] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
